docs/gettingStarted/customScheme/clientForNewScheme.md

```diff
@@ -0,0 +1,378 @@
+You might want to update client library while working on Alchemy integration if you added new contract or updated subgraph
```

Comment:
Needs a title and an Introduction section at the top

Reply:
Ahh its tricky! So title for this gets picked up from mkdocs and Intro is in previous file in this tutorial.

I wish I had better way of sharing how the site would look once published